src/app/zap-templates/templates/app/att-storage.zapt

```diff
@@ -13,11 +13,11 @@
 #define CLUSTER_MASK_DEFAULT_RESPONSE_FUNCTION (0x04)
 // Does this cluster have message sent function?
 #define CLUSTER_MASK_MESSAGE_SENT_FUNCTION (0x08)
-// Does this cluster have manufacturer specific attribute changed function?
-#define CLUSTER_MASK_MANUFACTURER_SPECIFIC_ATTRIBUTE_CHANGED_FUNCTION (0x10)
```

Comment:
This `CLUSTER_MASK_MANUFACTURER_SPECIFIC_ATTRIBUTE_CHANGED_FUNCTION` isn't used. Replacing it seems preferable to adding a new cluster mask, as we would need to change the mask to 16 bits